include/tables/dmadata/dmadata_table_us.h

```diff
@@ -0,0 +1,1563 @@
+/**
+ * Matching dmadata layout for US
```

Comment:
Should this specify version number or will the dma table be identical for us-region games?

Reply:
Tbh I'm not 100% sure. I can't imagine them changing file order, but I could be completely wrong. I added 1.0 to the comment header just to clarify.